app/ante/fixedgastx.go

```diff
@@ -0,0 +1,92 @@
+package ante
+
+import (
+	"github.com/Decentr-net/decentr/x/community"
+	"github.com/Decentr-net/decentr/x/pdv"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+type FixedGasTxDecorator struct {
+	config map[string]func(ctx sdk.Context) sdk.Gas
+}
+
+func NewFixedGasTxDecorator(pk pdv.Keeper, ck community.Keeper) FixedGasTxDecorator {
+	config := map[string]func(ctx sdk.Context) sdk.Gas{
+		pdv.MsgDeleteAccount{}.Type(): func(ctx sdk.Context) sdk.Gas {
```

Comment:
looks like it's better to use reflect.Type instead of string

Reply:
```golang
		if fixedGas, ok := ftd.config[msg.Type()]; ok {
			limit := ctx.GasMeter().Limit()
			consumed := fixedGas(ctx)
			return ctx.WithGasMeter(NewFixedGasMeter(consumed, limit)), nil
		}
```
Msg interface has a  Type() method, no need to use reflection.